terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars

```diff
@@ -0,0 +1,9 @@
+environment = "live"
+aws_profile = "live-eu-west-2"
+max_task_count = 8
+# service configs
+use_set_environment_files = true
+log_level = "debug"
```

Comment:
Are you sure you want to have log_level=debug in live?

Reply:
@JoelClemence We copied these configs from a live pipeline.

Reply:
@nawazish587 @JoelClemence I'm sure Joel is correct i think Debug might be too much for live so this can be changed to info. Whichever live application you seen that also used debug, you might want to notify the team to change it to info.

Reply:
thanks @JoelClemence @Ngriffiths1. I have changed it. I shall raise it with my team as sansaction search  has a debug level log. That may also be changed to info level.